Firmware/optiboot_w25x20cl.cpp

```diff
@@ -99,9 +99,9 @@ struct block_t;
 extern struct block_t *block_buffer;
 
 //! @brief Enter an STK500 compatible Optiboot boot loader waiting for flashing the languages to an external flash memory.
-void optiboot_w25x20cl_enter()
+uint8_t optiboot_w25x20cl_enter()
```

Comment:
please document the new return values:
`//! @return 1 if ...`

Reply:
updated